internal/biz/lookup.go

```diff
@@ -37,15 +37,36 @@ func (s *GetSubjectsUsecase) Get(ctx context.Context, req *v0.LookupSubjectsRequ
 	}
 
 	if req.Resource == nil {
+		s.log.WithContext(ctx).Infof("Missing Resource in request %v", req)
 		return nil, nil, errors.BadRequest("Invalid request", "Object is required")
 	}
 
 	if req.SubjectRelation != nil {
 		subjectRelation = *req.SubjectRelation
 	}
 
+	if req.SubjectType == nil {
+		s.log.WithContext(ctx).Infof("Missing SubjectType in request %v", req)
```

Comment:
I was thinking you could use `.Errorf` or whatever the error method is, since these are errors, but now that I think about it, there is no error in the API service (which is working properly), just in the incoming request, so perhaps `Info` is correct or `warning` as some people seem to use.
Interesting discussion: https://stackoverflow.com/questions/11497171/in-production-code-whats-the-log-level-when-failing-to-validate-request-parame

(This is just a comment, so feel free to mark as resolved when you are happy, whether you make a change or not.)